code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -685,7 +685,28 @@
 									step_away(V,location,15)
 						else
 							usr << "<span class = 'notice'>You were interrupted and couldn't fart! Rude!</span>"
+
 					lastFart=world.time
+
+					var/obj/item/weapon/storage/bible/B = locate(/obj/item/weapon/storage/bible) in src.loc
+					if(B)
+						if(!iscult(src))
+							src << "<span class='danger'>You feel incredibly guilty for farting on [B]!</span>"
+							if(prob(80)) //80% chance to escape God's justice

```

Comment:
You mean, 20%, since there's a 80% chance that will return nonzero and kill you?